pkg/task/task.go

```diff
@@ -1570,8 +1576,10 @@ func (m *executor) listOEV1alpha1URRaw(opts metav1.ListOptions) (result []byte,
 	uc := upgraderesult.KubeClient(upgraderesult.WithNamespace(m.getTaskRunNamespace()))
 	urList, err := uc.List(opts)
 	if err != nil {
+		err = errors.Wrap(err, "failed to list upgraderesult")
 		return
 	}
 	result, err = json.Marshal(urList)
+	err = errors.Wrapf(err, "failed to list upgraderesult: %s", urList.String())
```

Comment:
error check is missing

Reply:
The new `Wrapf` and `Wrap` functions return nil if the `err` is nil. But still added the error check.